docs: 1.9 upgrade guide (#3915)
medusajs/medusa: docs/content/upgrade-guides/medusa-core/1-9-0.md

---
description: 'Actions Required for v.1.9.0'
sidebar_custom_props:
  iconName: 'server-stack-solid'
---

# v1.9.0

Version 1.9.0 of Medusa introduces database schema changes which require running the migrations command. Additionally, it ships with a minor breaking change to the events system.

## Overview​

This release introduces a new column `metadata` on Sales Channels. It is required to run migrations to ensure your backend works as expected.

This release also comes with a breaking change to the events system. It is required to update your event bus packages to ensure events are handled correctly.

> In the core EventBusService, a check has been introduced to only insert staged jobs i.e. fire events, if there are subscribers to it.

Reviewer: ```suggestion
In the core `EventBusService`, a check has been introduced to only insert staged jobs (in other words, fire events), if there are subscribers to it.
```

---

## How to Update

Run the following command in the root directory of your Medusa Backend to update the core:

```bash npm2yarn
npm install @medusajs/medusa@1.9.0
```

> Update your event bus dependency. If you are using an official package, run one of these two commands:

Reviewer: ```suggestion
Then, update your event bus dependency. If you are using an official package, run one of these two commands based on the package you have installed:
```
